ui/tests/acceptance/replication/dr/secondary-test.js

```diff
@@ -0,0 +1,19 @@
+import { module, test } from 'qunit';
+import { later } from '@ember/runloop';
+import { setupApplicationTest } from 'ember-qunit';
+import { currentURL, visit, pauseTest } from '@ember/test-helpers';
```

Comment:
you'll want to make sure you remove `pauseTest` when you're done, and it looks like `later` isn't needed either :)